docs/howto/mirroring.md

```diff
@@ -0,0 +1,270 @@
+
+Mirroring in lakeFS allows replicating a lakeFS repository ("source") into read-only copies ("mirror") in different locations.
+
+Unlike conventional mirroring, data isn't simply copied between regions - lakeFS Cloud tracks the state of each commit, advancing the commit log on the mirror only once a commit has been fully replicated and all data is available.
+
+
+![mirroring architecture](../assets/img/mirroring/arch.png)
+
+
+## Uses cases
+
+### Disaster recovery
+
+Typically, object stores provide a replication/batch copy API to allow for disaster recovery: as new objects are written, they are asynchronously copied to other geo locations. 
+
+In the case of regional failure, users can rely on the other geolocations which should contain relatively-up-to-date state.
+
+The problem is reasoning about what managed to arrive by the time of disaster and what hasn't:
+* have all the necessary files for a given dataset arrived?
+* in cases there are dependencies between datasets, are all dependencies also up to date?
+* what is currently in-flight or haven't even started replicating yet?
+
+Reasoning about these is non-trivial, especially in the face of a regional disaster, However ensuring business continuity might require that we have these answers.
+
+Using lakeFS mirroring makes it much easier to answer: we are guaranteed that the latest commit that exists in the replica is in a consistent state and is fully usable, even if it isn't the absolute latest commit - it still reflects a known, consistent, point in time.
+
+
+### Data Locality
+
+For certain workloads, it might be cheaper to have data available in multiple regions: Expensive hardware such as GPUs might fluctuate in price, so we'd want to pick the region that currently offers the best pricing. The difference could easily offset to cost of the replicated data.
+
+The challenge is reproducibility - Say we have an ML training job that reads image files from a path in the object store. Which files existed at the time of training?
+
+If data is constantly flowing between regions, this might be harder to answer than we think. And even if we know - how can we recreate that exact state if we want to run the process again (for example, to rebuild that model for troubleshooting).
+
+Using consistent commits solves this problem - with lakeFS mirroring, it is guaranteed that a commit ID, regardless of location, will always contain the exact same data.
+
+We can train our model in region A, and a month later feed the same commit ID into another region - and get back the same results.
+
+
+## Setting up mirroring
+
+### Configuring bucket replication on S3
+
+The objects within the repository are copied using your cloud provider's object store replication mechanism.
+For AWS S3, please refer to the [AWS S3 replication documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-how-setup.html) to make sure your lakeFS repository's storage namespace is replicated.
```

Comment:
"to make sure your lakeFS repository's storage namespace is replicated."
Not clear enough. Possible customers questions: What's a storage namespace? What path should be the target?